.github/workflows/dapr.yml

```diff
@@ -173,3 +190,71 @@ jobs:
             --body "${RELEASE_BODY}" \
             --prerelease true \
             ${RELEASE_ARTIFACT[*]}
+      - name: Package Helm chart
+        env:
+              HELM_CHARTS_DIR: charts/dapr
+        run: |
+          if [ "$LATEST_RELEASE" = "true" ]; then
+            mkdir -p ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+            helm package ${{ env.HELM_CHARTS_DIR }} --destination ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+          fi
+      - name: Upload Helm charts package to artifacts
+        uses: actions/upload-artifact@master
+        with:
+          name: dapr_helm_charts_package
+          path: ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+  helmpublish:
+    name: Publish helm charts to Helm github pages repo
+    needs: publish
+    if: github.event_name != 'pull_request'
```

Comment:
`startswith(github.ref, 'refs/tags/v') and !contains(github.ref, '-rc.')` 

will detect it if it is the last build or not. then we do not need additional `if ` inside bash script.

Reply:
fixed